primitives/evm/src/lib.rs

```diff
@@ -131,9 +131,6 @@ impl WeightInfo {
 			(self.ref_time_usage, self.ref_time_limit)
 		{
 			let ref_time_usage = self.try_consume(cost, ref_time_limit, ref_time_usage)?;
-			if ref_time_usage > ref_time_limit {
-				return Err(ExitError::OutOfGas);
-			}
```

Comment:
Why delete this check in the hook? In my understanding, the proof record hostfunction cannot stop the transaction execution process inside the executor. It can only give us a more accurate proof size assumed size.

Reply:
It's redundant, you can find it in try consume method